components/button.css

```diff
@@ -0,0 +1,128 @@
+@import url(/stylesheets/style.css);
+
+
+.page-container{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+
+.container{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+}
+
+.btn{
+    padding: 0.8rem 2rem;
+    border-radius: 6px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+
+}
+
+.btn-primary{
+    color: white;
+    background-color: var(--primary-100);
+}
+
+.btn-secondary{
+    color: white;
+    background-color: var(--secondary-100);
+}
+
+.btn-danger{
+    color: white;
+    background-color: var(--danger);
+}
+
+.btn-success{
+    color: white;
+    background-color: var(--success);
+}
+
+.btn-warning{
+    color: white;
+    background-color: var(--warning);
+}
+
+.btn-link{
+    text-decoration: none;
+    color: white;
+    background-color: var(--link-color);
+    padding: 0.6rem 1rem;
+}
+
+.outlined-primary{
```

Comment:
You can use btn-outline-primary in this also it would be more readable to users.